docs/howtos/locally-published-components-in-fenix.md

```diff
@@ -0,0 +1,118 @@
+# Using locally-published components in Fenix
+
+Note: This is a bit tedious, so feel free to ask in slack if there's a better
```

Comment:
not sure how long we will be using this process, but at this point I'm thinking we need a script that does all the steps below. Otherwise it is indeed tedious.

Reply:
I'm working on fixing the process and killing non-megazords after I finish the iOS import process.

This might still be needed in some cases, though.

Reply:
yeap! just to confirm, these are very good docs to have!

Reply:
It's hard to do via a script robustly, due to all the steps like 'edit this kotlin file to say this thing'.

I have considered writing one though.